appium/tests/screenobjects/splash.screen.ts

```diff
@@ -156,7 +156,7 @@ class SplashScreen extends BaseScreen {
         this.clickContinueBtn();
         this.changeLanguage();
         this.gmailLogin(email, password);
-        this.signInAsGoogleAccounLabel.waitForDisplayed({reverse: true});
+        this.signInAsGoogleAccounLabel.waitForDisplayed({timeout: 10000, reverse: true});
```

Comment:
If various different screens are being switched on the way, you may need to try/wait/retry as I suggested to avoid the flakiness.